tests/test_gpu_stump.py

```diff
@@ -53,7 +53,6 @@ def test_gpu_stump_int_input():
 
 
 @cuda.jit("(f8[:, :], f8[:], i8[:], i8, b1, i8[:])")
-@patch("stumpy.config.STUMPY_THREADS_PER_BLOCK", TEST_THREADS_PER_BLOCK)
```

Comment:
Kinda curious why you had to remove this?

Reply:
Yea so I was getting a "`cuda` has no property `grid`" error, I guess `@cuda` and `@patch` were messing with each other. But then I also noticed that it's not a test case anyway (as it does not begin with `test_`) so I removed it. This function is executed inside of a test case, so I don't think we would need to mock the config value here.